gcc/doc/extend.texi

```diff
@@ -21042,7 +21042,7 @@ processors.
 Returns the value that is currently set in the @samp{tp} register.
 @end deftypefn
 
-@subsection CORE-V Built-in Functions
+CORE-V Built-in Functions
```

Comment:
I think this needs to be
```
@node CORE-V Built-in Functions
@subsection CORE-V Built-in Functions
```
(`@node` will link for the info format output,`@subsection` for other formats.)

You'll also need to add a line to the menu section further up this document (see line 14,635).

Reply:
All fixed in latest commit